php_phongo.c

```diff
@@ -957,36 +957,36 @@ bool phongo_execute_command(mongoc_client_t* client, php_phongo_command_type_t t
 		goto cleanup;
 	}
 
-	/* According to mongoc_cursor_new_from_command_reply(), the reply bson_t
-	 * is ultimately destroyed on both success and failure. */
+	/* According to mongoc_cursor_new_from_command_reply_with_opts(), the reply
+	 * bson_t is ultimately destroyed on both success and failure. */
 	if (bson_iter_init_find(&iter, &reply, "cursor") && BSON_ITER_HOLDS_DOCUMENT(&iter)) {
 		bson_t       initial_reply = BSON_INITIALIZER;
 		bson_error_t error         = { 0 };
 
 		bson_copy_to(&reply, &initial_reply);
 
 		if (command->max_await_time_ms) {
-			bson_append_bool(&initial_reply, "awaitData", -1, 1);
-			bson_append_int64(&initial_reply, "maxAwaitTimeMS", -1, command->max_await_time_ms);
-			bson_append_bool(&initial_reply, "tailable", -1, 1);
+			bson_append_bool(&opts, "awaitData", -1, 1);
+			bson_append_int64(&opts, "maxAwaitTimeMS", -1, command->max_await_time_ms);
+			bson_append_bool(&opts, "tailable", -1, 1);
 		}
 
 		if (command->batch_size) {
-			bson_append_int64(&initial_reply, "batchSize", -1, command->batch_size);
+			bson_append_int64(&opts, "batchSize", -1, command->batch_size);
 		}
 
-		if (zsession && !mongoc_client_session_append(Z_SESSION_OBJ_P(zsession)->client_session, &initial_reply, &error)) {
+		if (zsession && !mongoc_client_session_append(Z_SESSION_OBJ_P(zsession)->client_session, &opts, &error)) {
```

Comment:
Another reason to split `opts` from a new `cursor_opts` document: this is currently appending explicit sessions twice to the same document.